path.cabal

```diff
@@ -1,95 +1,193 @@
+cabal-version:       3.0
 name:                path
 version:             0.9.5
 synopsis:            Support for well-typed paths
 description:         Support for well-typed paths.
-license:             BSD3
+license:             BSD-3-Clause
 license-file:        LICENSE
 author:              Chris Done <chrisdone@fpcomplete.com>
 maintainer:          Chris Done <chrisdone@fpcomplete.com>
 copyright:           2015–2018 FP Complete
 category:            System, Filesystem
 build-type:          Simple
-cabal-version:       1.18
-tested-with:         GHC==8.6.5, GHC==8.8.4, GHC==8.10.1
+tested-with:         GHC==9.2.8, GHC==9.4.8, GHC==9.6.6, GHC==9.8.2, GHC==9.10.1
 extra-source-files:  README.md
                    , CHANGELOG
+                   , os-string-compat/System/OsString/Compat/Include.hs
                    , src/Path/Include.hs
                    , src/Path/Internal/Include.hs
+                   , src/OsPath/Include.hs
+                   , src/OsPath/Internal/Include.hs
                    , test/Common/Include.hs
+                   , test-ospath/Common/Include.hs
+                   , validity-test-ospath/Include.hs
+                   , validity-test-ospath/OsPath/Gen/Include.hs
 
 flag dev
   description:        Turn on development settings.
   manual:             True
   default:            False
 
+flag os-string
+  description:        Use an older version of the os-string library.
+  manual:             False
+  default:            False
+
```

Comment:
This doesn't make sense to me.
The default is `False`, but if you turn on the flag it becomes .. `False`?

Reply:
Oh I think I completely misunderstood what `manual` means. Then you can ignore this comment.

Reply:
Ok, good! Referencing the relevant documentation here anyway: https://cabal.readthedocs.io/en/stable/cabal-package-description-file.html#pkg-field-flag-manual